biopandas/mmcif/pandas_mmcif.py

```diff
@@ -479,3 +491,51 @@ def read_mmcif_from_list(self, mmcif_lines):
         # self.header, self.code = self._parse_header_code()
         self.code = self.data["entry"]["id"][0].lower()
         return self
+
+    def get_pandas_pdb(self, offset_chains: bool = True, records: List[str] = ["ATOM", "HETATM"]) -> PandasPdb:
```

Comment:
should this perhaps be called `convert_to_pandas_pdb?` Just so that it doesn't indicate it's fetching the PDB